### DISCUSSION TITLE
Joomla 5 / Astroid 3.16 - change color of one menu option

### DISCUSSION THREAD

**Discussion by cloudguy23** [templaza/astroid-framework#832] [Q&A]
Hello.   I'm using Joomla 5 and Astroid v3.16.   I want to change the background of a single menu option.   TBH, I don't know where to start.   I've tried to apply a custom CSS to the Joomla Menu option but it doesn't seem to override my global Astroid Config.   All I'm looking to accomplish is change the global color and background for one menu option, giving it the illusion of a "button":

.astroid-nav .nav-link {
    color: rgb(255, 255, 255);
    background-color: rgb(1, 177, 136);
}

Thank you.

**Comment by otherway-fr** [comment]
Hi,
This can be easily achieved without CSS tricking.
Open the menu item you want to emphasize.
In Astroid menu options "Custom class" enter ie **text-bg-success** 
Save, it's done.
You can use all of bootstrap classes.

![screen_20241017_111422](https://github.com/user-attachments/assets/40c57efd-b19e-4e95-9ad1-3c7b23048b0b)
